guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>You can register your data as a big data file share using the <code>add_bigdata()</code> method on a <code>DatastoreManager</code> object. Ensure the datasets are stored in a format compatible with the GeoAnalytics server, as demonstrated earlier in this guide.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>